app/src/main/java/nerd/tuxmobil/fahrplan/congress/preferences/SharedPreferencesRepository.kt

```diff
@@ -113,4 +114,11 @@ class SharedPreferencesRepository(val context: Context) {
         putInt(ENGELSYSTEM_SHIFTS_HASH_KEY, hash)
     }
 
+    fun getSelectedSessionId() =
+        preferences.getString(SELECTED_SESSION_ID_KEY, "")!!
```

Comment:
Is the newline between `=` and `preferences....` by choice?

Reply:
Yes. To have a consistent formatting with the surrounding functions.